lndr-cli/test/Spec.hs

```diff
@@ -368,23 +364,22 @@ verifySettlementTest = do
                                                         (Just settleAmountInWei)
                                                         Nothing
     let txHash = fromRight (error "error sending eth") txHashE
+        creditRecord = CreditRecord testAddress4
+                                    testAddress1
+                                    10
+                                    ""
+                                    testAddress4
+                                    0
+                                    ""
+                                    ""
+                                    ucacAddr
+                                    (Just $ unQuantity settleAmountInWei)
+                                    (Just "ETH")
+                                    (Just 0)
```

Comment:
**Non-blocking feedback:** `CreditRecord` has 12 positional arguments, making it very difficult to grok L367-378. Generally, if a function takes more than one argument, and some arguments are optional or can be defaulted, and the ordering matters, and the ordering is not intuitive, then either the function should be refactored or the arguments should instead be *named arguments* (i.e: a hash/map/key-value pairs).